crates/handlers/src/views/logout.rs

```diff
@@ -39,6 +41,22 @@ pub(crate) async fn post(
                     .record_browser_session(&clock, &session)
                     .await;
 
+                // First, get RP-initiated logout endpoints before actually finishing the
```

Comment:
We need to make sure that this logic also happens when the logout is done through the React frontend. I don't have a good suggestion on how to do so for now though

Reply:
I am looking on how to integrate this [here](https://github.com/tchapgouv/matrix-authentication-service/blob/9b06223c1df940c15da512dd56659b4aa3dece57/crates/handlers/src/graphql/mutations/browser_session.rs#L87) but I am not sure this is the way to go